src/copy.h

```diff
@@ -3,9 +3,34 @@
 
 #include <postgres.h>
 #include <nodes/parsenodes.h>
+#include <access/xact.h>
+#include <executor/executor.h>
+#include <commands/copy.h>
 
 typedef struct Hypertable Hypertable;
+typedef struct ChunkDispatch ChunkDispatch;
+typedef struct CopyChunkState CopyChunkState;
 
-Oid			timescaledb_DoCopy(const CopyStmt *stmt, const char *queryString, uint64 *processed, Hypertable *ht);
+typedef bool (*CopyFromFunc) (CopyChunkState *ccstate, ExprContext *econtext,
+							  Datum *values, bool *nulls, Oid *tuple_oid);
+
+typedef struct CopyChunkState
```

Comment:
I don't think the internals of this struct need to be public. Also we have `typedef struct CopyChunkState CopyChunkState;`